crates/ide-db/src/assists.rs

```diff
@@ -29,7 +29,16 @@ pub struct Assist {
     /// cumbersome, especially if you want to embed an assist into another data
     /// structure, such as a diagnostic.
     pub source_change: Option<SourceChange>,
-    pub trigger_signature_help: bool,
+    /// The command to execute after the assist is applied.
+    pub command: Option<Command>,
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum Command {
+    /// Show the parameter hints popup.
+    TriggerSignatureHelp,
+    /// Rename the just inserted item.
+    Rename,
```

Comment:
bikeshedding: A better name might be something like `TriggerRename`, to match the naming of the trigger parameter signature help command.

Reply:
The trigger verb used here seems like it stemmed from the name of the editor action "editor.action.triggerParameterHints". This runs "editor.action.rename", so I'm leaning towards leaving it as rename (but happy to change it if needed)